sdk/include/opentelemetry/sdk/logs/logger_provider_factory.h

```diff
@@ -24,7 +24,7 @@ namespace logs
 /**
  * Factory class for LoggerProvider.
  */
-class LoggerProviderFactory
+class OPENTELEMETRY_EXPORT LoggerProviderFactory
```

Comment:
Do we need also export log processor factories?

Reply:
Yes, the all the log processor factories need to be exported. The `SimpleLogRecordProcessorFactory` class was in the `openteleemtry_cpp.def` file which makes the `__declspec(export)` unnecessary. But I added the export declaration to all the log record factories. Thanks for catching.